docs/contributing/README.md

```diff
@@ -15,7 +15,7 @@ To contribute changes to the code or easyconfigs, you should:
 
 0) Create and setup a [GitHub account](https://github.com/join) (and register your SSH public key);
 
-1) First fork and then also clone the appropriate GitHub repository, for example:
+1) First clone and fork the appropriate GitHub repository, for example:
```

Comment:
Original order is correct, you fork first, then clone

Reply:
Logically, yes. If done on the web, yes. But as it's done on the command line in the example, the order doesn't exactly do that. One clones the origin repo and then push it to ones own namespace. That being said, I doubt anyone will notice the discrepancy, but it irked me.

Reply:
Hmm, you are right.

#boegel Isn't this getting complicated since now `origin` is the main repo and not their fork? It doesn't really match what the contribution process we actually suggest in the docs: https://easybuild.readthedocs.io/en/latest/Contributing.html#forking-the-repository

Reply:
> Hmm, you are right.

I'm as surprised as anyone.

> @boegel Isn't this getting complicated since now origin is the main repo and not their fork? It doesn't really match what the contribution process we actually suggest in the docs: https://easybuild.readthedocs.io/en/latest/Contributing.html#forking-the-repository

Doesn't `git checkout -b develop origin/develop` insinuate that origin is `easybuild/easyconfigs` or similar? At least for me, I have always used `origin` as a label for the original repository, and used my username (or something similar) for the fork.

Reply:
That doesn't map to what comes next in that page: https://easybuild.readthedocs.io/en/latest/Contributing.html#keeping-the-develop-branch-in-sync
where `upstream` is clearly the main repo, and `origin` is the users fork. To make sure `develop` tracks upstream by default you can use
```
git branch --set-upstream-to=upstream/develop develop
```

Reply:
Ah, great catch! God, how did we end up having a conversation about git workflow semantics. I want my day back.

Reply:
`origin` usually points to the repo at `easybuilders` for me too.

We should fix the docs here imho, it's fine as is in the tutorial.

Reply:
Also, for this part of the tutorial it doesn't matter what they call `origin`, since they're using `--new-pr` (which creates temporary remotes in a temporary directory by itself, it doesn't rely on `origin` being one or the other)